solutions/swb-app/src/generateRouter.ts

```diff
@@ -104,7 +103,6 @@ export function generateRouter(apiRouteConfig: ApiRouteConfig): Express {
   setUpUserRoutes(router, apiRouteConfig.userManagementService);
   setUpEnvTypeRoutes(router, apiRouteConfig.environmentTypeService);
   setUpEnvTypeConfigRoutes(router, apiRouteConfig.environmentTypeConfigService);
-  setUpProjectRoutes(router, apiRouteConfig.projectService);
```

Comment:
Was this intentional?

Reply:
I pulled in some of Marianna's code that added projects functionality. Removed all those references but if this existed before then I'm going to have to do some sort of diff between my feature branch and develop :sigh: